stable/minio/README.md

```diff
@@ -108,8 +108,8 @@ The following table lists the configurable parameters of the MinIO chart and the
 | `existingSecret`                          | Name of existing secret with access and secret key.                                                                                     | `""`                                       |
 | `accessKey`                               | Default access key (5 to 20 characters)                                                                                                 | `AKIAIOSFODNN7EXAMPLE`                     |
 | `secretKey`                               | Default secret key (8 to 40 characters)                                                                                                 | `wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY` |
-| `configPath`                              | Default config file location                                                                                                            | `~/.minio`                                 |
-| `configPathmc`                            | Default config file location for MinIO client - mc                                                                                      | `~/.mc`                                    |
+| `configPath`                              | Default config file location                                                                                                            | `/etc/minio`                                 |
```

Comment:
`--config-dir` has been deprecated from MinIO server as of minio/minio#7033, can you please remove this option completely?

Reply:
I've removed the option. Haven't implemented the certs-dir functionality at this point though.

Are you expecting --certs-dir to be implemented under this ticket? That seems out of scope for what I was updating and I'm not sure how to set up any tests to handle it. Doesn't look like anyone raised the deprecation as an issue either.